content/docs/command-reference/run.md

```diff
@@ -6,14 +6,17 @@ command.
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
-               [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
-               [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
-               [--plots <path>] [--plots-no-cache <path>] [--live <path>]
+usage: dvc run [-h] [-q | -v] [-n <name>] [-f]
+               [-d <path>] [-p [<filename>:]<params_list>]
+               [-o <filename>] [-O <filename>] [-c <filename>]
+               [--external] [--outs-persist <filename>]
+               [--outs-persist-no-cache <filename>]
+               [-m <path>] [-M <path>]
+               [--plots <path>] [--plots-no-cache <path>]
+               [--no-exec] [--no-commit] [--no-run-cache]
+               [--always-changed] [--live <path>]
```

Comment:
My preference would be to start by copying what you have below for `dvc stage add` and then append the additional options (`--no-exec`, `--no-commit`, --no-run-cache`) at the very end. What do you think?

Reply:
Good call. Standardized in cb88988e.